test/empty-img-src.md

```diff
@@ -0,0 +1,2 @@
+
+
```

Comment:
This may fail once the CI runs since (https://github.com/Alir3z4/html2text/commit/d51fb1dfb66774d4db339e03c452676cd73d44da) it doesn't append a new line on the generated markdown,

Let's run the CI and see 🤞

Reply:
CI passed, seems I was wrong.